torch/csrc/jit/passes/symbolic_shape_analysis.cpp

```diff
@@ -31,7 +31,7 @@ XXX: this is still in prototype phase and has much work left to do, including
 but not limited to:
 - Refactor APIs
 - Add decent coverage of common ops
-- Add shape analysis pass on Graph that handles Ifs and Loops
+- Add shape analysis pass on Graph that handles Loops
```

Comment:
you don't seem to be removing if handling in this PR. why change this comment?

Reply:
It was done already and i forgot to remove this line